02_activities/assignments/assignment.sh

```diff
@@ -8,17 +8,18 @@ set -x
 # > Add your code below each comment to complete the tasks
 
 # 1. Create 5 directories named dir1, dir2, dir3, dir4, and dir5
-
+mkdir dir1 dir2 dir3 dir4 dir5
 # 2. List the contents of the parent directory to verify the presence of the 5 directories
-
+ls
 # 3. Create 5 text files in dir2 named file1, file2, file3, file4, and file5
-
+touch ./02_activities/assignments/assignment.sh/dir2 file1.txt file2.txt file3.txt file4.txt file5.txt
```

Comment:
Q3. Your code seems to creat the five files, but you will also create a file name `./02_activities/assignments/assignment.sh/dir2` and it is not guarantee that your file is created at the right directory you wanted.